azure-pipelines.yml

```diff
@@ -1,5 +1,57 @@
-pool:
-  vmImage: 'Ubuntu 16.04'
-steps:
-- script: echo Hello, world!
-  displayName: 'Hello World'
+
+jobs:
+  - job: build_docker_images
+    displayName: Build Docker Images
+    condition: not(eq(variables['Build.SourceBranch'], 'refs/heads/master'))
+    pool:
+      vmImage: 'ubuntu-16.04'
+    steps:
+    - script: docker build -t aswfstaging/aswf-vfx-base:2019 -f aswf-vfx-base/Dockerfile .
+      displayName: Build aswf-vfx-base
+    - script: docker build -t aswfstaging/aswf-ocio-build:2019 -f aswf-ocio-build/Dockerfile .
+      displayName: Build aswf-ocio-build
+    - script: docker build -t aswfstaging/aswf-openvdb-build:2019 -f aswf-openvdb-build/Dockerfile .
+      displayName: Build aswf-openvdb-build
```

Comment:
Perhaps we can simplify these docker names? I wouldn't mind something like aswf/openvdb:2019 ?

Reply:
Indeed! I used the `-build` postfix as it seems to be the standard for build images. What about something like `aswf/ci-openvdb`? I would interpret `aswf/openvdb` as a docker image that contains openvdb itself...

Reply:
I see what you mean about using aswf/openvdb, so yeah perhaps aswf/ci-openvdb is better.